language/unit_tests/test_client.py

```diff
@@ -115,11 +116,31 @@ def test_document_from_url_factory_explicit(self):
 
         encoding = Encoding.UTF32
         gcs_url = 'gs://my-text-bucket/sentiment-me.txt'
-        document = client.document_from_url(gcs_url, doc_type=Document.HTML,
-                                            encoding=encoding)
+        document = client.document_from_gcs_url(gcs_url,
+                                                doc_type=Document.HTML,
+                                                encoding=encoding)
         self.assertIsInstance(document, Document)
         self.assertIs(document.client, client)
         self.assertIsNone(document.content)
         self.assertEqual(document.gcs_url, gcs_url)
         self.assertEqual(document.doc_type, Document.HTML)
         self.assertEqual(document.encoding, encoding)
+
+    def test_document_from_url_deprecation(self):
+        import warnings
+
+        creds = make_mock_credentials()
+        client = self._make_one(credentials=creds, http=object())
+
+        Client = self._get_target_class()
+        with mock.patch.object(Client, 'document_from_gcs_url') as dfgu:
```

Comment:
Yes, I know it is a short variable name. I would prefer to keep it because I find it easier to read than a line wrap.